packages/volto/src/components/theme/Navigation/Navigation.jsx

```diff
@@ -26,7 +26,7 @@ const Navigation = (props) => {
   const intl = useIntl();
   const dispatch = useDispatch();
   const { pathname, type } = props;
-  const [isMobileMenuOpen, setisMobileMenuOpen] = useState(false);
+  const [isMobileMenuOpen, setisMobileMenuOpen] = useState(null);
```

Comment:
Please change the default value of isMobileMenuOpen back to false; this change isn't needed.